README.md

```diff
@@ -8,11 +8,12 @@ Private WireGuard® networks made easy
 
 This repository contains all the open source Tailscale client code and
 the `tailscaled` daemon and `tailscale` CLI tool. The `tailscaled`
-daemon runs primarily on Linux; it also works to varying degrees on
-FreeBSD, OpenBSD, Darwin, and Windows.
+daemon runs on Linux, various BSDs, macOS, iOS, Android, and Windows.
```

Comment:
The `tailscaled` daemon does not run on iOS or Android.

Also, this loses the subtlety that `tailscaled` does not run as well on FreeBSD, OpenBSD, etc. We have lots of missing pieces on those operating systems.